.eslintrc.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
         // Allows the use of 'void' operator for promises we deliberately don't wait for,
         // providing a way to bypass the 'no-floating-promises' rule when necessary.
         'no-void': ['error', { allowAsStatement: true }],
+        "@typescript-eslint/no-explicit-any": "warn"
```

Comment:
warn or error? I think tsc is erroring after the config change right?

Reply:
for now, warn, once we fix the issues, we can change it to warn (see other related tickets)